exotica/cmake/Exotica.cmake

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 
-message(STATUS "Compiling using c++ 11 (required by EXOTica)")
-add_compile_options(-std=c++11)
+# message(STATUS "Compiling using c++ 11 (required by EXOTica)")
+# add_compile_options(-std=c++11)
```

Comment:
Remove unused code

Reply:
Which is the entire file, and then you need to remove CFG_EXTRAS from the main CMakeLists.txt